app/uk/gov/hmrc/agentaccesscontrol/model/PayeUtr.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentaccesscontrol.model
+
+import uk.gov.hmrc.domain.TaxIdentifier
+
+case class PayeUtr(value: String) extends TaxIdentifier
```

Comment:
My hunch is that there is no such thing as a PAYE UTR and we should use EmpRef (probably already in the domain lib). Let's see what Hamish replies on Slack.